src/pybacked.rs

```diff
@@ -152,7 +152,7 @@ impl From<Bound<'_, PyBytes>> for PyBackedBytes {
         let b = py_bytes.as_bytes();
         let data = NonNull::from(b);
         Self {
-            storage: PyBackedBytesStorage::Python(py_bytes.to_owned().unbind()),
+            storage: PyBackedBytesStorage::Python(py_bytes.clone().unbind()),
```

Comment:
I'm not sure about this one - maybe `to_owned` is more readable?

Reply:
The lint description says that there is no difference, and `to_owned` is misleading because there is no change in ownership.